functions/commonfn.php

```diff
@@ -248,11 +250,15 @@ function amuRegisterFromForm() {
 */
 
 class amuUserObject{
-		
+
+    private $config;
```

Comment:
Is this variable used? It is being flagged as unused in my IDE so suspect it might not be. Would you be able to take a look at this please Richard? Cheers

Reply:
removed this. config has been replaced by site options, so this is no longer necessary

Reply:
Cheers. Looks good. Merging in!